README.md

```diff
@@ -121,6 +121,8 @@ options:
   -v, --verbose         print more verbose logs (you can repeat `-v` to make it more verbose)
   --stdin-display-name STDIN_DISPLAY_NAME
                         the name used when processing input from stdin
+  --config-file CONFIG_FILE
```

Comment:
Can you make it just `--config` and add tests?

Reply:
done!